server.js

```diff
@@ -1,98 +1,138 @@
 'use strict';
-//allows us to use the config method to access our .env file
-require('dotenv').config();
-// the library we use in node to set up our server
+
 const express = require('express');
+require('dotenv').config();
 const cors = require('cors');
-//port number from .env file
-const PORT = process.env.PORT || 3000;
 const app = express();
-//weather data from json file
-const weatherData = require('./data/weather.json');
-// middleware
 app.use(cors());
 
+// const axios = require('axios');
```

Comment:
Hey there, you are on your way here, so nice work!

You will need this line (line 9) so that you can make a request using axios to grab the weather data.